src/components/Form.js

```diff
@@ -61,8 +61,7 @@ export default class Form extends Component {
 
   onChange = (formData, options={validate: false}) => {
     const liveValidate = this.props.liveValidate || options.validate;
-    const errors = liveValidate ? this.validate(formData) :
-                                  this.state.errors;
+    const errors = liveValidate ? this.validate(formData) : [];
```

Comment:
Addressed in https://github.com/mozilla-services/react-jsonschema-form/pull/177/commits/19aa552945c2b5ebf976623fc2eca8c25478e4aa